### PR TITLE
Bug/#7 doxygens section headings are not getting displayed

### LINKED ISSUE #36: Create project description at PyPi
The project description at PyPi is currently empty. Deploy readme.

Ref: https://packaging.python.org/en/latest/guides/making-a-pypi-friendly-readme/

### PR DESCRIPTION
**Pull request by mb-emag**
Beside the section headings this also tries to fix #36 (as this bugfix will create a new release i wanted to add the readme fix for pypi beforehand).

### CODE REVIEW

File: demo/demo/src/readme.md [hunk 1]
```diff
@@ -1,5 +1,28 @@
...
+
+> If you use markdown pages in doxygen and want to reference e.g. classes you can use the special syntax
+> provided by doxygen-markdown (see <https://doxygen.nl/manual/markdown.html>), e.g.
+> [This is a link to the Car class](@ref doxysphinx::rst::Car).
+
+### Linking to sections
+
+You can also reference the doxygen sections manually like this (using final html document names):
+
+* [Namespace List](namespaces.html)
```
Comment (twodrops) [inline demo/demo/src/readme.md:23]: @mb-emag Do we have some information in the documentation (not as am example) which answers the question
> How to make Doxysphinx sections like pages, modules, namespaces, annotated, files appear in the generated documentation?
Answer is imply to include the doxysphinx generated `index.rst` in a `toctree` 

Sorry, I was too lazy to search the documentation for this and raise a PR.

**Review by twodrops — CHANGES_REQUESTED** [review]

File: demo/demo/src/readme.md (hunk 1 above)
Comment (twodrops) [reply demo/demo/src/readme.md:23]: @mb-emag Ping, incase the notification was missed and you are waiting for me :)

Comment (mb-emag) [reply demo/demo/src/readme.md:23]: @twodrops, sorry i missed that one.

Yes we have it documented here: <https://boschglobal.github.io/doxysphinx/docs/linking_to_doxygen.html#via-document-link>

**Comment by twodrops** [comment]
No problem. Do you we have specific topic which talks about making those Doxygen subchapters visible? I could not find it under the above link

File: demo/demo/src/readme.md (hunk 1 above)
Comment (twodrops) [reply demo/demo/src/readme.md:23]: @mb-emag No problem. Do you we have specific topic which talks about making those Doxygen subchapters visible? I could not find it under the above link

Comment (mb-emag) [reply demo/demo/src/readme.md:23]: Ah ok guess it's too short then, i mean't this sentence here:
> This is typically used **to reference the root docs or some special page where you know the exact name** (Don’t use it for C++ Symbols, as there are better ways to link them

However, beside that we have no documentation how to link to the namespace pages, related pages section etc.
Perhaps we could add an info box right below the "inside a toctree directive" on that page where we name these standard doxygen sections? or an faq entry?

Comment (mb-emag) [reply demo/demo/src/readme.md:23]: I'll do it with this one :-)

Comment (mb-emag) [reply demo/demo/src/readme.md:23]: @twodrops - I added a tip section with the standard pages we have in the doxysphinx repo. I'm not sure what pages doxygen could have in an addition to that... should we add other ones too? (you can see the ones I've added here: https://github.com/boschglobal/doxysphinx/pull/45/commits/149b5e252ebff0a960832440d522ea9323472011)

**Pull request merged**